src/app/layout.tsx

```diff
@@ -32,4 +32,6 @@ export const metadata: Metadata = {
     description: "Providing foodbank services to clients in south London.",
 };
 
+export const dynamic = "force-dynamic";
```

Comment:
try removing this and see if it still works? main has this implemented in _just_ the PDF downloader... which hopefully fixes things!